framework/packages/abstract-client/src/client.rs

```diff
@@ -19,6 +19,10 @@ impl<Chain: CwEnv> AbstractClient<Chain> {
         Ok(Self { abstr })
     }
 
+    pub fn ans_host(&self) -> &AnsHost<Chain> {
```

Comment:
Call this `name_service`?
```suggestion
    pub fn name_service(&self) -> &AnsHost<Chain> {
```